### PR TITLE
Update download.go

### PR DESCRIPTION
**Pull request by mija**

### CODE REVIEW

**Comment by kepler471** [comment]
Will take a look at this later, thanks for finding the problem!
What is your setup and environment, is ffmpeg on your path or is the executable in the working directory?
I have checked my output logs and have not seen the same error.

**Comment by mija** [comment]
I first tried the windows .exe but it didn't work at all so I instead installed go and went ahead using that.
First I fetched all the necessary stuff, discordgo etc etc then i included C:/Go in my environment variable for %PATH% and made sure to keep ffmpeg.exe and youtube-dl.exe there.

The only way I could make it function was to create a folder with the name of the video I was attempting to attach in discord and then put youtube-dl.exe inside that folder. in that case it would work, but this clearly wasnt an okay workaround since id have to manually throw in youtube-dl.exe into each folder created, or copy it over there each time the folder is made.

hopefully my explanation was good enough.

Basically Go is installed on C:\Go and i run it with "go run dvembed". I manually fetched all files I needed, not sure why I ended up doing it manually but I did. Updated env variable to make sure both Go Path and Path (wasnt entirely sure if %PATH% within Go would refer to "Go Path") included C:\Go

**Comment by kepler471** [comment]
Thanks for the breakdown.

It looks like youtube-dl.exe is not properly added to the Windows path. I am not using Windows for the bot, but from past experience I know I've had trouble with the path variables. It shouldn't be necessary to add an executable to every directory.

I will also need to test that all the Go directory functionality is working properly on Windows, this is something I did not try. No paths are hardcoded in strings so it should be fine but worth checking. I am currently running it on Linux, so I will look into how to build this for Windows properly. One thing to add will be to add the ".exe" tag on the commands.

**Comment by kepler471** [comment]
Looking at why the .exe. was not working, I think that was because the executables were old and had not included a lot of changes.
